demo/starter.js

```diff
@@ -0,0 +1,44 @@
+/* eslint no-console: "off" */
+const { exec } = require("child_process");
+const { Action } = require("tasit-sdk");
+const { ConfigLoader } = Action;
+const { ProviderFactory } = require("tasit-action/dist/ProviderFactory");
+const tasitSdkConfig = require("./config/default.js");
+
+const CONSOLE_FG_RED = "\x1b[31m";
+const CONSOLE_RESET = "\x1b[0m";
```

Comment:
What are these strings?

Reply:
These strings are used to change `console.log` color output (https://stackoverflow.com/a/41407246)

Reply:
Cool. Thanks!